packages/gatsby/src/commands/develop-process.ts

```diff
@@ -80,7 +80,7 @@ const openDebuggerPort = (debugInfo: IDebugInfo): void => {
 }
 
 module.exports = async (program: IDevelopArgs): Promise<void> => {
-  reporter.setVerbose(program.verbose)
+  reporter.setVerbose(isTruthy(process.env.VERBOSE) || program.verbose)
```

Comment:
I realized that this isn't complete - we should set `program.vebose` to true if env var is truthy because of things like https://github.com/gatsbyjs/gatsby/blob/a8f7101e1ff6f1b437c44d78bb9f3eafa81dc539/packages/gatsby/src/commands/develop-process.ts#L135-L137 which check `if (program.verbose) ` before doing some potentially expensive things